apps/dashboard/src/contract-ui/tabs/nfts/components/claim-form.tsx

```diff
@@ -79,29 +88,32 @@ export const NFTClaimForm: React.FC<NFTClaimFormProps> = ({ contract }) => {
               action: "claim",
               label: "attempt",
             });
-            claim.mutate(
-              { quantity: d.amount, to: d.to },
-              {
-                onSuccess: () => {
-                  trackEvent({
-                    category: "nft",
-                    action: "claim",
-                    label: "success",
-                  });
-                  onSuccess();
-                  modalContext.onClose();
-                },
-                onError: (error) => {
-                  trackEvent({
-                    category: "nft",
-                    action: "claim",
-                    label: "error",
-                    error,
-                  });
-                  onError(error);
-                },
+
+            const transaction = claimTo({
+              contract,
+              to: d.to ?? "",
+              quantity: BigInt(d.amount),
```

Comment:
V4 was able to take no `to` field. not sure what's changed in v5

Reply:
@jnsdls can probably verify this one

Reply:
it's an extension so ig it is what it is (we have to specify the field)

Reply:
`claimTo` _has_ to have an address, the behavior in v4 for claim without address would claim to the connected address so we have to do the same here, empty string will throw an error during the mutation here I believe

Reply:
unless we handle this in the extension (we shouldn't!) the consumer is responsible to pass the address here (meaning dashboard has to pass it) `d.to` is no longer optional here

Reply:
correct. just read the code for getClaimParams. I believe if we dont pass it we will get an "execution reverted"

Reply:
yep, so form should be invalid here if no address is filled out meaning we should never need an  empty string for this case

Reply:
let me check. i think typescript was screaming at me (irrc it didn't understand the required* field from react-form-hook which should prevent the falsy case)

Reply:
if that's the case just `if(!d.to) {don't try to do the tx just show the error immediately here}`